File: app/src/main/java/com/wealoha/social/beans/Comment2DTO.java
package com.wealoha.social.beans;

import com.wealoha.social.beans.UserDTO;

public class Comment2DTO {

	public long createTimeMillis;
	public UserDTO user;
	public boolean mine;
	public String id;
	public String type;
	public String comment;
	public UserDTO replyUser;
	public boolean whisper;
}

File: app/src/main/java/com/wealoha/social/beans/Comment2DTO.kt
package com.wealoha.social.beans

data class Comment2DTO(
    var createTimeMillis: Long = 0,
    var user: UserDTO,
    var mine: Boolean = false,
    var id: String? = null,
    var type: String? = null,
    var comment: String,
    var replyUser: UserDTO,
    var whisper: Boolean = false
) {

}